### PR TITLE
Update node dependencies for Angular apps

### PR DESCRIPTION
**Pull request by pvliss**
Update to Angular 7.2.7 and other minor and patch releases

In continuation of #9333 

Relates to #9285 

Please make sure the below checklist is followed for Pull Requests.

-   [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/angular/package.json.ejs [hunk 1]
```diff
@@ -27,24 +27,25 @@
...
+    "@angular/core": "7.2.7",
+    "@angular/forms": "7.2.7",
+    "@angular/platform-browser": "7.2.7",
+    "@angular/platform-browser-dynamic": "7.2.7",
+    "@angular/router": "7.2.7",
     "@fortawesome/angular-fontawesome": "0.3.0",
     "@fortawesome/fontawesome-svg-core": "1.2.15",
     "@fortawesome/free-solid-svg-icons": "5.7.2",
     "@ng-bootstrap/ng-bootstrap": "4.0.4",
+    "@nguniversal/express-engine": "7.1.0",
```
Comment (wmarques) [inline generators/client/templates/angular/package.json.ejs]: Why is this added?

Comment (pvliss) [reply generators/client/templates/angular/package.json.ejs]: Oh sorry, forgot to comment about this one. It is a peer dependency of `ngx-cookie` and needs to be added manually. See also here https://github.com/salemdar/ngx-cookie/issues/78. This has changed in this commit: https://github.com/salemdar/ngx-cookie/commit/1956eff8940859147268f5b2b6cc3e7619a89247. Alternatively we can revert back to the previous version

Comment (wmarques) [reply generators/client/templates/angular/package.json.ejs]: I don't understand why we should add this... The main purpose of this dependency is to do server-side rendering

Comment (pvliss) [reply generators/client/templates/angular/package.json.ejs]: You are right. It should not be needed. Let me check again

Comment (pvliss) [reply generators/client/templates/angular/package.json.ejs]: So you were right, it was not needed at all. I removed it. Lets wait for the CI again

For some reason it was failing locally and the error message was the same as the one described here https://github.com/salemdar/ngx-cookie/issues/78 and hastily I applied one of the workarounds without checking what the dep is actually for. Good thing we have review :)

Comment (pvliss) [reply generators/client/templates/angular/package.json.ejs]: @wmarques So it appears that the problem is that webpack fails to build the bundle as can be seen at the CI logs after commit cf2bfad.Tried to silence webpack by using the [IgnorePlugin](https://webpack.js.org/plugins/ignore-plugin/) in commit 8115e8c but this is no good at runtime :(. So we either wait for a fix for ngx-cookie or revert back to 4.0.2, except if you have some better idea/solution to this problem. What do you think?

Comment (wmarques) [reply generators/client/templates/angular/package.json.ejs]: Let's revert to 4.0.2 and wait for a fix. If it's not fixed in few weeks we can think about dropping this, I don't link being dependent on a non-maintained lib...

Comment (pvliss) [reply generators/client/templates/angular/package.json.ejs]: :+1:
